src/bridge/bridge.ts

```diff
@@ -1,23 +1,56 @@
 import { ethers } from 'ethers';
 
-import { assetUnitsToDecimal, decimalToPip } from '#pipmath';
+import { assetUnitsToDecimal, decimalToPip, multiplyPips } from '#pipmath';
 
 import { getExchangeAddressAndChainFromApi } from '#client/rest/public';
-import {
-  ExchangeStargateAdapter__factory,
-  IPool__factory,
-  IStargateFeeLibrary__factory,
-  IStargateRouter__factory,
-} from '#typechain-types/index';
-import { BridgeTarget } from '#types/enums/request';
+import { ExchangeStargateV2Adapter__factory } from '#typechain-types/factories/contracts/bridge-adapters/ExchangeStargateV2Adapter.sol/ExchangeStargateV2Adapter__factory';
+import { IStargateV2__factory } from '#typechain-types/index';
+import { StargateV2Target } from '#types/enums/request';
```

Comment:
@dileepfrog here as well - doesnt seem to be an issue to me?

For future i added the typechain-types to the blacklist so it will ignore changes to them in its reviews.